lib/takeaway.rb

```diff
@@ -1,10 +1,28 @@
 require_relative 'menu.rb'
 
 class Takeaway
+attr_reader :cart
+
+  def initialize
+    @cart = []
+  end
```

Comment:
What is the responsibility of the Takeaway class? One might be tempted to create a whole different class that looks after the 'cart' (or the order of the customer), thereby delegating the responsibility of storing the selection